cmd/ocm-support/patch/organizations/cmd.go

```diff
@@ -0,0 +1,87 @@
+package organizations
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	v1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-online/ocm-support-cli/cmd/ocm-support/utils"
+	"github.com/openshift-online/ocm-support-cli/pkg/organization"
+	"github.com/openshift-online/ocm-support-cli/pkg/request"
+)
+
+var args struct {
+	filter     string
+	dryRun     bool
+	maxRecords int
+}
+
+// CmdPatchOrganizations represents the organizations patch command
+var CmdPatchOrganizations = &cobra.Command{
+	Use:     "organizations [id]",
+	Aliases: utils.Aliases["organizations"],
+	Short:   "Patches organizations matching the filter",
+	Long:    "Patches organizations matching the filter",
+	RunE:    run,
+	Args:    cobra.ExactArgs(1),
+}
+
+func init() {
+	flags := CmdPatchOrganizations.Flags()
+	flags.BoolVar(
+		&args.dryRun,
+		"dryRun",
+		true,
+		"If false, it will execute the patch command call in instead of a dry run.",
+	)
+	flags.IntVar(
+		&args.maxRecords,
+		"maxRecords",
+		utils.MaxRecords,
+		"Maximum number of affected records. Only effective when dryRun is set to false.",
+	)
+}
+
+func run(cmd *cobra.Command, argv []string) error {
+	var organizationsToPatch []*v1.Organization
+	connection, err := ocm.NewConnection().Build()
+	if err != nil {
+		return fmt.Errorf("failed to create OCM connection: %v", err)
+	}
+	filter := argv[0]
+	// by default, returns all organizations found
+	size := -1
+	organizationsToPatch, err = organization.GetOrganizations("", filter, size, false, false, true, connection)
+	if err != nil {
+		return err
+	}
```

Comment:
what would happen if we pass an empty filter, like:

`ocm support patch orgs ""`

Reply:
It would try and patch all organizations. Added a non-empty validation